src/internal/confluent/client.go

```diff
@@ -16,10 +16,13 @@ import (
 )
 
 type CloudApiAccess struct {
-	ApiEndpoint     string
-	Username        string
-	Password        string
-	UserApiEndpoint string
+	ApiEndpoint                 string
+	Username                    string
+	Password                    string
+	UserApiEndpoint             string
+	StreamGovernanceApiEndpoint string
+	StreamGovernanceApiUsername string
+	StreamGovernanceApiPassword string
```

Comment:
The necessity of this surprises me, as we could (iirc) create topics before.
That is a different endpoint than the one we need to query them?

Reply:
It's for schemas rather than topics. From what I can gather the endpoint is used for Schema Registry and Stream Catalog and is called the Stream Governance API. I don't believe you can hit ListSchemas on the same endpoint as topics, however please verify in-case I have overlooked something?